apps/vault/package.json

```diff
@@ -34,21 +33,22 @@
   ],
   "license": "GPL-3.0-or-later",
   "description": "",
-  "devDependencies": {
-    "@aragon/apps-shared-migrations": "1.0.0",
-    "@aragon/cli": "^6.0.0",
-    "@aragon/test-helpers": "^2.1.0",
-    "@aragon/truffle-config-v4": "^1.0.1",
-    "eth-ens-namehash": "^2.0.8",
-    "eth-gas-reporter": "^0.2.0",
-    "ethereumjs-testrpc-sc": "^6.5.1-sc.0",
-    "ganache-cli": "^6.4.3",
-    "solidity-coverage": "0.6.2",
-    "solium": "^1.2.3",
-    "truffle": "4.1.14",
-    "truffle-extract": "^1.2.1"
-  },
   "dependencies": {
+    "@aragon/apps-vault": "^4.1.0",
     "@aragon/os": "4.2.0"
+  },
+  "devDependencies": {
+    "@aragon/buidler-aragon": "^0.2.0-beta.2",
+    "@aragon/contract-test-helpers": "0.0.1",
+    "@nomiclabs/buidler": "^1.0.2",
+    "@nomiclabs/buidler-etherscan": "^1.2.0",
+    "@nomiclabs/buidler-solhint": "^1.2.0",
+    "@nomiclabs/buidler-truffle5": "^1.1.2",
+    "@nomiclabs/buidler-web3": "^1.1.2",
+    "bignumber.js": "^9.0.0",
+    "buidler-gas-reporter": "^0.1.3",
+    "eth-ens-namehash": "^2.0.8",
+    "solidity-coverage": "^0.7.0-beta.3",
+    "web3": "^1.2.6"
```

Comment:
I think this is a new dependency—do we need to include it explicitly (e.g. buidler expects us to install this ourselves for the tests)?

Reply:
This is a peer dependency of `@nomiclabs/buidler-web3` so a requirement to be installed as well. If at some point we move all code from web3 to ethers on our plugin then we can stop adding these dependencies.